### DISCUSSION TITLE
Followed course well but at the end i dont see a  GitHub Pages visibility dropdown menu.

### DISCUSSION THREAD

**Discussion by LornaJoyce** [skills/.github#337] [Communicate using Markdown] [answered]
Followed course well but at the end i dont see a  GitHub Pages visibility dropdown menu. I see the following Visibility GITHUB ENTERPRISE
With a GitHub Enterprise account, you can restrict access to your GitHub Pages site by publishing it privately. A privately published site can only be accessed by people with read access to the repository the site is published from. You can use privately published sites to share your internal documentation or knowledge base with members of your enterprise.

[Learn more about the visibility of your GitHub Pages site](https://docs.github.com/pages/getting-started-with-github-pages/changing-the-visibility-of-your-github-pages-site)

**Comment by sinsukehlab** [comment, marked as answer]
It seems that you have tried to publish `/docs`. The repository doesn't have such a directory. Select `/ (root)` from the **Select folder** drop-down menu.

**Reply by LornaJoyce** [reply]
Thanks! Works now

**Reply by sinsukehlab** [reply]
I'm glad that I could be of help.
